calculator_app/src/components/Calculator.js

```diff
@@ -0,0 +1,244 @@
+import React, { Component } from "react";
+import "./css/Calculator.css";
+
+class Calculator extends Component {
+  state = {
+    displayValue: "0",
+    previousValue: null,
+    operator: null,
+    waitingNewVal: false
+  };
+
+  handleNumClicks = (e) => {
+    const { displayValue, waitingNewVal } = this.state;
+    if (displayValue === "0" || waitingNewVal === true) {
+      this.setState({
+        displayValue:  e.target.value 
+      });
+    } else
```

Comment:
Missing the `{ }` brackets  - that error prevented numbers larger than single digit numbers appear on the screen.